airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SimpleLeftSide/SimpleLeftSide.tsx

```diff
@@ -0,0 +1,64 @@
+import { faGoogle } from "@fortawesome/free-brands-svg-icons";
+import { faCheckCircle, faEnvelope } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useState } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { HeadTitle } from "components/common/HeadTitle";
+import { Button } from "components/ui/Button";
+import { FlexContainer } from "components/ui/Flex";
+import { Heading } from "components/ui/Heading";
+
+import styles from "./SimpleLeftSide.module.scss";
+import { OAuthLogin } from "../../../OAuthLogin";
+import { Disclaimer, SignupForm } from "../SignupForm";
+
+const Detail: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => {
+  return (
+    <FlexContainer gap="sm" alignItems="center" className={styles.detailTextContainer}>
+      <FontAwesomeIcon icon={faCheckCircle} className={styles.checkIcon} />
+      {children}
+    </FlexContainer>
+  );
+};
+export const SimpleLeftSide: React.FC = () => {
+  const [showOauth, setShowOauth] = useState(true);
```

Comment:
:broom: Once we clean this up, it would be nicer to manage this via an URL route instead. Currently when switching between those the back button of your browser won't go back, since we only manage this state in JS, which already got me multiple times. I think for the final implementation this shoudl be separate routes, so your back button works as expected when switching between them.

Reply:
I'm curious, if it's the same action (signup) but just two different flows, in which cases do you think it make more sense to have two routes?
I didn't think about the back button to be honest, as you can just use the elements in the page to navigate, but I'm talking from my ignorance in this matter!